cypress/integration/seededFlows/adminFlows/displayAds/displayAds.spec.js

```diff
@@ -0,0 +1,87 @@
+describe('Display Ads', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginAndVisit(user, '/admin/customization/display_ads');
+
+      cy.findByRole('table').within(() => {
+        cy.findByRole('button', { name: 'Destroy' }).click();
+      });
+    });
+  });
+
+  describe('delete a display ad', () => {
+    it('should display confirmation modal', () => {
+      cy.findByRole('dialog').contains('Confirm changes').should('be.visible');
+    });
+
+    it('should display warning text if confirmation text does not match', () => {
+      cy.findByRole('dialog').within(() => {
+        cy.get('input').type('Text that does not match.');
+        cy.findByRole('button', { name: 'Confirm changes' }).click();
+
+        cy.get('.crayons-notice')
+          .contains('The confirmation text did not match.')
+          .should('be.visible');
+
+        cy.get('button[aria-label="Close"]').click();
```

Comment:
suggestion (not blocking) 
```suggestion
        cy.findByRole('button', { name: "Close" }).click();
```